src/Compatibility/Microsoft.DotNet.PackageValidation/GetLastStablePackage.cs

```diff
@@ -37,7 +37,7 @@ protected override void ExecuteCore()
                 SourceRepository repository = Repository.Factory.GetCoreV3(nugetFeed);
                 FindPackageByIdResource resource = repository.GetResource<FindPackageByIdResource>();
                 SourceCacheContext cache = new SourceCacheContext();
-                IEnumerable<NuGetVersion> versions = resource.GetAllVersionsAsync(PackageId, cache, NullLogger.Instance, CancellationToken.None).Result;
+                IEnumerable<NuGetVersion> versions = resource?.GetAllVersionsAsync(PackageId, cache, NullLogger.Instance, CancellationToken.None).Result;
```

Comment:
If resource is null, will any of logic below actually do anything? If not, should you just stop execution at this point instead?

Reply:
resource is not null, i will remove it.